CHANGES/1033.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Normalize default ports according to :rfc:`3986`.
+
+Specified ports are removed from the str representation of an URL if the port matches
+the schemes default port. -- by :user:`commonism`.
```

Comment:
If you want the byline to be in the same paragraph, don't put a period before it.
```suggestion
the schemes default port -- by :user:`commonism`.
```

Alternatively, you could attach it to the very first sentence, too. Another possibility is to drop the trailing period and add an empty line as a separator so it's not a part of the sentence and the paragraph.